packages/core/src/utils/pushNotification.ts

```diff
@@ -0,0 +1,86 @@
+import axios from 'axios';
+import config from '../config';
+import { client as prismic } from '../utils/prismic';
+import { NotificationType } from '../interfaces/app/appNotification';
+import { models } from '../database';
+import { AppUserModel } from '../database/models/app/appUser';
+import localesConfig from '../utils/locale.json';
+
+export async function sendNotification(
+    users: AppUserModel[],
+    type: NotificationType,
+) {
+    // registry notification
+    await models.appNotification.bulkCreate(users.map(el => ({
+        userId: el.id,
+        type,
+        isWallet: true,
+        isWebApp: true,
+    })));
+
+    // filter users that have walletPNT or appPNT
+    users = users.filter(el => el.walletPNT || el.appPNT);
+
+    // get only unique languages
+    const languages = [...new Set(users.map(el => el.language))];
+
+    // get notification from prismic
+    const prismicNotifications = {};
+    const promises = languages.map(async language => {
```

Comment:
```suggestion
    const fetchMessagesFromPrismic = languages.map(async language => {
```

Suggesting this naming because I first got confused with what was happening here and was about to comment something very non-sense 🤣